src/js/redux/reducers/search/searchFiltersReducer.js

```diff
@@ -6,13 +6,15 @@
 import { Set } from 'immutable';
 
 import * as AwardFilterFunctions from './filters/awardFilterFunctions';
+import updateSelectedLocations from './filters/locationFilterFunctions';
```

Comment:
See previous comment. Let's change this to a namespaced wildcard `import` to be consistent with how other such imports are handled in the codebase.